docs/ingress.md

```diff
@@ -119,7 +119,19 @@ spec:
            number: 8000
 ```
 
-The above ingress resource will create an ApiDefinition inside Tyk's Gateway. The ApiDefinition will offer path-based routing listening on /httpbin. Because the referenced template is `myapideftemplate`, the IngressReconciler will retrieve the `myapideftemplate` resource and determine that the ApiDefinition object it creates needs to have standard auth enabled.
+Tyk Ingress Controller will create APIs in Tyk for each path defined for a specific rule in Ingress resource. Each API 
+created inside Tyk will follow a special naming convention as follows:
+```
+<ingress_namespace>-<ingress_name>-<hash(Host + Path)>
```

Comment:
It will be useful to mention which hash method are we using.

Reply:
I think it's sha256

Reply:
Updated in https://github.com/TykTechnologies/tyk-operator/pull/488/commits/5f010da012a78387696c45d29f261cc5e579b089